### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by TheDarkonion** [ThatOneHidde/thatonehidde.github.io#1]
actually, this isnt a issue report i wanted to ask you something so, im trying to make a MyDrugs site of HTSDOF the netflix series but i wanted to ask you if i could use your code because i saw your website its so good i also wanted to make such a Shop-System but i dont know how to thats the website i meant: [https://mydrugsto.netlify.app/](url) 
Its so good and i really want that shop system and the "SHOP NOW BUTTON" css code and html it would be very nice.

**Comment by ThatOneHidde** [comment]
Thanks for reaching out to me.
The site that is up now, is just a static website (w/o any functionality), some time ago i rewrote the whole thing in vuejs and coded a whole backend with it. Maybe i will release the source soon. Can you explain me what you would do with my code?

**Comment by TheDarkonion** [comment]
Yes, im making a recreate of MyDrugs site. Because i saw your site and i wanted to make something like that. and i saw your shop page, its really good. I want this Shop thing like with the user rating and border and also do it in my code. Because my shop doesnt really look good most ideas i have is inspired by you i also give credits if you want so. I also can remove them, im sorry that i didnt asked you sooner. i would appreciate that. also my website is already published: [https://thedarkonion.github.io/MyDrugs/](url)

**Comment by TheDarkonion** [comment]
to make it clearer, i wanna use some of your code to better my page (Shop-now button, shop (border, ecstasy) ) because mine arent that good i only have one type of ecstasy and also no container.

**Comment by ThatOneHidde** [comment]
I think it is a nice way of learning HTML and CSS by recreating other sites. You can always use the devtools to see how elements are constructed. I will probably release the full shop system on GitHub in the future. Good luck with your project.

**Comment by TheDarkonion** [comment]
Hey, thank you a lot it helped me out a lot really man. I appreciate it very much thank you for your help.
